src/__docs__/breakpoint.mdx

```diff
@@ -0,0 +1,174 @@
+---
+name: Breakpoint
+route: /breakpoint
+menu: Components
+---
+
+
+# Intent
+
+The Breakpoint module provides information about the current viewport size, or the size of a specific element. It watches for changes to either of these, and renders your component with new props.
+
+Managing breakpoints from JS gives you full control over the view. You can render totally different components for different sizes. Additionally, you can pass a breakpoint object to child components without them knowing what the actual numbers are; only the names of each breakpoint.
+
+This module can also do breakpoints on any element in the DOM; it's not restricted to `window`. This means your component doesn't have to assume it'll be used in any specific container. The component's concern is that it renders appropriately for the space it's given.
+
+## Usage
+
+With the high order component variant, you receive one prop which is the 'bp' object. It has methods such as `props.bp.isGte('medium')`.
+
+With the render callback variant, you receive 'bp' as the first argument to the function.
+
+They can be imported like this:
+
+```javascript
+import { BreakpointHoc } from 'rearm/lib/Breakpoint';
+// or the render variant
+import { BreakpointRender } from 'rearm/lib/Breakpoint';
+```
+You configure it by providing an array of breakpoint objects containing constraints.
+
+If you don't provide `minWidth`, it defaults to `0`. If you don't proivide `maxWidth`, it defaults to `Infinity`.
+
+```javascript
+const MyComponent = ({ bp }) => (
+  bp.isLt('medium') ? <MobileView /> : <DesktopView />
+);
+const breakpoints = [
+  { name: 'small', maxWidth: 600 },
+  { name: 'medium', minWidth: 601, maxWidth: 1199 },
+  { name: 'large', minWidth: 1200, exact: true },
+];
+export default BreakpointHoc({ breakpoints, type: 'viewport' });
+```
+
+Or use the render callback variant:
+
+```javascript
+const MyComponent = () => (
+  <BreakpointRender breakpoints={breakpoints} type="viewport">
+    {bp => bp.isLt('medium') ? <MobileView /> : <DesktopView />}
+  </BreakpointRender>
+);
+export default MyComponent;
+```
+
+Here are the outputs of all of the bp methods given these breakpoints. If you resize your browser, they will change.
+
+```
+isLt('small')false
```

Comment:
These, of course, do not change. Could we add back the live example? I assume you'd make a separate file for the documentation components (BreakpointDocsHelpers.js or something) and import/use them here.